config/dev.env

```diff
@@ -0,0 +1,4 @@
+PORT=3000
```

Comment:
move the file to root and rename to `.env.dev`
load `.env` via https://www.npmjs.com/package/dotenv

Reply:
Fixed.
But how will this work if I need different environements? For example for tests.